components/content.py

```diff
@@ -15,7 +15,9 @@ def get_component():
             children=[
                 html.H1(id="h1", children="Plando-prototype"),
                 dbc.Row(
-                    children=[ActiveFilters.get_component(), Notes.get_component()]
+                    style={"width": "60vw"},
+                    className="row p-1 justify-content-start",
```

Comment:
Change `className="row p-1 justify-content-start",`
To `className="justify-content-start",`